ipatests/prci_definitions/gating.yaml

```diff
@@ -21,316 +21,316 @@ topologies:
     memory: 2750
 
 jobs:
-  fedora-latest/build:
+  fedora-latest-ipa-4-12/build:
     requires: []
     priority: 150
     job:
       class: Build
       args:
         git_repo: '{git_repo}'
         git_refspec: '{git_refspec}'
-        template: &ci-master-latest
-          name: freeipa/ci-master-f39
+        template: &ci-ipa-4-12-latest
+          name: freeipa/ci-ipa-4-12-f40
           version: 0.0.1
```

Comment:
shouldn't be:

`version: 0.0.0 ?`

![image](https://github.com/freeipa/freeipa/assets/43062603/b032b120-c48b-459b-a483-39796304f033)

Reply:
@f-trivino 
the latest version is 0.0.1:
![image](https://github.com/freeipa/freeipa/assets/21310191/067232d4-eef9-43fe-b07a-69c268203d0d)

Reply:
I see it now, thanks